theme/palette.ts

```diff
@@ -25,6 +25,9 @@ export const lightPalette = {
   edge: "rgba(255, 255, 255, 0.15)",
 
   ...paletteShared,
+
+  danger: "#FF3B30",
```

Comment:
What's the difference between danger and red?
Also since these are the same I think they should go into the shared palette
cc @thierryskoda

Reply:
Maybe ask Andrew if adding a new "red" color was intended, otherwise yes we can just reuse the original red we had